services/stsafea/stsafea_asymmetric_key_slots.c

```diff
@@ -231,9 +231,9 @@ stse_ReturnCode_t stsafea_query_generic_public_key_slot_info(
 	if(*pPresence_flag == 1)
 	{
 		stse_ecc_key_type_t curve_id_index;
-		*pKey_type = 0;
+		*pKey_type = (stse_ecc_key_type_t)0;
```

Comment:
Would it be possible to modify the logic to use STSE_ECC_KT_INVALID instead of casting a constant ?

Reply:
Initialization to STSE_ECC_KT_INVALID done